Composer/packages/ui-plugins/teams/src/triggers.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { UISchema } from '@bfc/extension-client';
+import { SDKKinds } from '@bfc/shared';
+import formatMessage from 'format-message';
+
+export default {
+  [SDKKinds.OnAppBasedLinkQuery]: {
+    trigger: {
+      submenu: {
+        label: formatMessage('Microsoft Teams'),
```

Comment:
It might be better to define constants before this object (e.g. `const TEAMS_LABEL = formatMessage('Microsoft teams')`) and then use those for messages that repeat a lot.